docs/tutorials/fundamentals/viewer.md

```diff
@@ -341,39 +295,58 @@ viewer.add_image(blobs, name='blobs', opacity=0.5, colormap='red')
 nbscreenshot(viewer, alt_text="A 2d view of the moon on top of which is overlaid a 3d volume containing blobs through which you can navigate using the dimension slider.")
 ```
 
-In order to get or to update the current position of the slider, use:
+In order to get or update the current position of the slider, use:
 
-```python
-# to get the current position
+```{code-cell} python
```

Comment:
This also needs to be changed back. Again, it is not supposed to be executed.
```suggestion
```python
```